web-components/src/components/inputs/new/ConfirmationCode/ConfirmationCode.tsx

```diff
@@ -119,22 +119,18 @@ export const ConfirmationCode = forwardRef<
 
     const handleOnPaste = (e: React.ClipboardEvent<HTMLInputElement>) => {
       const pastedValue = e.clipboardData.getData('Text');
-
       let currentInput = 0;
 
       for (let i = 0; i < pastedValue.length; i++) {
         const pastedCharacter = pastedValue.charAt(i);
-        const currentValue = inputsRef.current[currentInput].value;
         if (pastedCharacter.match(inputProps.pattern)) {
-          if (!currentValue) {
```

Comment:
code change is just about removing this check so that we set the code value with the newly pasted code even if there's already one
the diff is larger because of the indent change